web/styles/dark_theme.css

```diff
@@ -536,8 +536,7 @@
     .subscriber-list-box .subscriber_list_container .subscriber-list td,
     .member-list-box,
     .member-list-box .member_list_container .member-list td,
-    #subscription_overlay .settings-radio-input-parent,
-    #recent_view_table table td {
```

Comment:
`recent_view_row` has `td` selectors.

Reply:
Right but there is no `#recent_view_table table td`.

You think it is supposed to be `#recent_view  table td`?

Reply:
There is `#recent_view_table` in `index.html`.

Reply:
Right and in `web/src/recent_view_ui.ts` we set the elements content:
```
    const rendered_body = render_recent_view_body({
        search_val: $("#recent_view_search").val() ?? "",
        ...get_recent_view_filters_params(),
    });
    $("#recent_view_table").html(rendered_body);
```
And there is no `td` element in `web/templates/recent_view_table.hbs`

Reply:
It is `const $container = $("#recent-view-content-tbody");` in `web/src/recent_view_ui.ts` where inside which we render `recent_view_row.hbs`. And as we can see from `index.html`:
```
<div class="recent_view_container">
       <div id="recent_view_table"></div>
</div>
<table id="recent-view-content-table">
      <tbody data-empty="{{ _('No conversations match your filters.') }}" id="recent-view-content-tbody"></tbody>
</table>
```

I don't think there is any `td` element in `#recent_view_table` element.

Reply:
oh yeah! You are right that we should replace it to use `#recent_view`.

Reply:
I don't think we need to as `recent_view.css` already applies border-color:
```
 & td {
        vertical-align: middle;
        padding: 3px 8px;
        border-top: 1px solid var(--color-border-recent-view-row);
    }
```

Where `-color-border-recent-view-row: hsl(0deg 0% 0% / 20%);` has the same value as the defined in `dark_theme.css`.